Services/Repository/Favourites/templates/default/tpl.favourites.html

```diff
@@ -0,0 +1,2 @@
+<script src="Services/Dashboard/Block/js/ReplaceModalContent.js"></script>
+<div>{FAV_CONTENT}</div>
```

Comment:
The only purpose of this template seems to be to integrate a js from the Dashboard service, please move this template to the Dashboard service, too.

Reply:
The repository component should not be responsible to load dashboard javascript code. This violates SoC principles. This results in high coupling and bad maintainability. As I wrote I understand that there is currently no clear interface, but these lines may produce severe issues in the future. A Dashboard maintainer should be able to change or even delete this JS file without the need to check other components.